components/notification/index.en-US.md

```diff
@@ -0,0 +1,56 @@
+---
+category: Components
+type: Views
+noinstant: true
+english: Notification
+---
+
+To display a notification message globally.
+
+## When to use
+To display a notification message at the top right of the view port. Typically it can be
+used in the following case:
+
+- A notification with complext content.
+- A notification of interaction with users, typically, it will provide next step hint
```

Comment:
- A notification providing a feedback based on the user interaction. Or it may show some details about upcoming steps the user may have to follow.